nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/pom.xml

```diff
@@ -52,49 +52,34 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-kerberos-user-service-api</artifactId>
+            <artifactId>nifi-iceberg-common</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-security-kerberos-api</artifactId>
+            <artifactId>nifi-kerberos-user-service-api</artifactId>
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-security-kerberos</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <artifactId>nifi-security-kerberos-api</artifactId>
```

Comment:
`nifi-security-kerberos-api` does not seem to be needed in this module. I cannot see class imports, the module compiles and the tests run without it.
I think it can be removed.